src/fty_srr.cc

```diff
@@ -55,94 +59,102 @@ void setSignalHandler()
     sigaction(SIGINT, &sigIntHandler, NULL);
 }
 
+/**
+ * Set Signal handler
+ */
+void terminateHandler()
+{
+    log_error((AGENT_NAME + std::string(" Error")).c_str());
+    exit(EXIT_FAILURE);
+}
```

Comment:
If we're going to implement a custom `std::terminate` handler, we need to do it right:
```c++
#include <stdexcept>
#include <fty_common_logging_library.h>
#include <cxxabi.h>

std::string demangle(const std::string &mangled) {
    int status = 0;
    char *buff = __cxxabiv1::__cxa_demangle(mangled.c_str(), nullptr, nullptr, &status);
    if (buff) {
        std::string demangled(buff);
        std::free(buff);
        return demangled;
    }
    else {
        return "<error>";
    }
}

void terminateHandler() {
    if (std::current_exception()) {
        try {
            std::rethrow_exception(std::current_exception());
        }
        catch (std::exception &e) {
            log_fatal("std::terminate called after unhandled exception of type '%s': '%s'.", demangle(__cxxabiv1::__cxa_current_exception_type()->name()).c_str(), e.what());
        }
        catch (...) {
            log_fatal("std::terminate called after unhandled exception of type '%s'.", demangle(__cxxabiv1::__cxa_current_exception_type()->name()).c_str());
        }
    }
    else {
        log_fatal("std::terminate called.");
    }
    exit(EXIT_FAILURE);
}

int main() {
    std::set_terminate(terminateHandler);
    throw std::runtime_error("Bwaa");
}
```
```
$ g++ test.cpp -std=c++11 -lfty_common_logging
$ ./a.out 
2019-10-28 10:06:06 ftylog [FATAL] test.cpp:24 std::terminate called after unhandled exception of type 'std::runtime_error': 'Bwaa'.
$
```
This is hefty enough to warrant inclusion in a `fty-common-*` library. Also, we should probably call `abort()` instead of `exit(EXIT_FAILURE)` in our custom terminate handler.